### PR TITLE
Feature forgot password

### LINKED ISSUE #10: Add "Forgot Password" Feature to Login Page
Implement a "Forgot Password" link on the login page. This link should direct users to a password recovery process, where they can enter their registered email address to receive instructions for resetting their password. This enhancement will improve user accessibility and overall experience.

### PR DESCRIPTION
**Pull request by sadath2001**
## What does this PR do?

Sends out an email to the user to reset his password along with a link to do so.
The user can reset his password using that link and can get back to his account
validations in place to check if the email exisits

Fixes #10 

## How should this be tested?

- Click forgot password
-  Add email to send the details
- check your email, reset password and try logging in with new password

### Required

- [X] Filled out the "How to test" section in this PR
- [X] Read Contributing guide
- [X] Self-reviewed my own code
- [X] Checked for warnings, there are none
- [X] Removed all `console.logs`
- [X] The changes don't cause any responsiveness issues

### CODE REVIEW

File: backend/Controllers/UserController.js [hunk 1]
```diff
@@ -107,23 +103,22 @@ const Login = async (req, res) => {
...
       .json({ success: false, message: "Internal Server Error" });
   }
-
-}
+};
 
 async function Sendcontactmail(req, res) {
   const { name, email, message, rating } = req.body; // Capture rating from the request
   console.log(req.body);
   try {
```
Comment (AlfiyaSiddique) [inline backend/Controllers/UserController.js:119]: remove this console,log @sadath2001

**Comment by AlfiyaSiddique** [comment]
@sadath2001 one comment rest looks good to me

**Review by sadath2001 — COMMENTED** [review]
Removed the console.log

**Comment by sadath2001** [comment]
hi @AlfiyaSiddique , kindly review

**Comment by AlfiyaSiddique** [comment]
@sadath2001 fix merge conflicts

**Review by sadath2001 — COMMENTED** [review]
hi @AlfiyaSiddique , kindly review

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**